db/memtable.cc

```diff
@@ -669,6 +669,12 @@ static bool SaveValue(void* arg, const char* entry) {
     if (!s->CheckCallback(seq)) {
       return true;  // to continue to the next seq
     }
+    if (ts_sz > 0) {
+      assert(s->key);
+      if (s->callback_ && !s->callback_->IsVisible(seq)) {
+        return true;  // to continue to the next seq
+      }
+    }
```

Comment:
I think you can remove this as well since `s->CheckCallback` above does the same thing.

Reply:
Right, I did the same for `GetContext::SaveValue`, but forgot this. Will update.